packages/docs-page/server/generate-static-props.ts

```diff
@@ -0,0 +1,154 @@
+import moize, { Options } from 'moize'
+
+import {
+  fetchNavData,
+  fetchDocument,
+  fetchVersionMetadataList,
+} from '../content-api'
+import { ENABLE_VERSIONED_DOCS, DEFAULT_PARAM_ID } from './consts'
+
+import renderPageMdx from '../render-page-mdx'
+
+import { stripVersionFromPathParams, normalizeVersion } from '../util'
+
+import { resolveNavData } from './resolve-nav-data'
+
+const moiseOpts: Options = { isPromise: true, maxSize: Infinity }
```

Comment:
good catch here

Reply:
need to change `moiseOpts` to `moizeOpts` 😂